stock_package_info/models/product_packaging_template.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from openerp.addons import decimal_precision as dp
+
+
+class ProductPackagingTemplate(models.Model):
+    _name = 'product.packaging.template'
+    _description = 'Product Packaging Template'
+    _rec_name = 'packaging_template_name'
+
+    packaging_template_name = fields.Char(required=True)
+    package_type = fields.Selection([
+        ('unit', 'Unit'),
+        ('pack', 'Pack'),
+        ('box', 'Box'),
+        ('pallet', 'Pallet'),
+    ],
+        required=True,
+    )
+    product_tmpl_id = fields.Many2one(
+        comodel_name='product.template',
+        string='Product',
+        index=True,
+        ondelete='cascade',
+    )
+    length = fields.Float(
```

Comment:
Although I'm wondering if we should bite the bullet in this particular instance. Renaming this will create a conflict with the `product.packaging` model that it is templating- https://github.com/odoo/odoo/blob/fc2e80cb4bcc450762c7ac5cb82a3e2d88062b38/addons/delivery/models/product_packaging.py#L12